src/extremeweatherbench/metrics.py

```diff
@@ -53,8 +64,10 @@ class RegionalRMSE(Metric):
     def compute(self, forecast: xr.DataArray, observation: xr.DataArray):
         rmse_values = []
         for init_time in forecast.init_time:
-            init_forecast, subset_observation = self.align_datasets(
-                forecast, observation, pd.Timestamp(init_time.values).to_pydatetime()
+            init_forecast, subset_observation = utils.temporal_align_dataarrays(
```

Comment:
Noting that the way you've packaged `utils.temporal_align_dataarrays` here is how I'd expect to see the previous `_align_observations_temporal_resampled` implemented.